package.json

```diff
@@ -1,123 +1,16 @@
-  "bugs": "https://github.com/eclipse/sprotty/issues",
-  "author": {
-    "name": "Eclipse Sprotty"
-  },
-  "contributors": [
-    {
-      "name": "Marc Dumais",
-      "email": "marc.dumais@ericsson.com",
-      "url": "https://www.ericsson.com/"
-    },
-    {
-      "name": "Takashi Katou"
-    },
-    {
-      "name": "Jan Köhnlein",
-      "email": "jan.koehnlein@typefox.io",
-      "url": "http://typefox.io"
-    },
-    {
-      "name": "Philip Langer",
-      "email": "planger@eclipsesource.com",
-      "url": "https://www.eclipsesource.com"
-    },
-    {
-      "name": "Tobias Ortmayr",
-      "email": "tortmayr@eclipsesource.com",
-      "url": "https://www.eclipsesource.com"
-    },
-    {
-      "name": "Miro Spönemann",
-      "email": "miro.spoenemann@typefox.io",
-      "url": "http://typefox.io"
-    }
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/eclipse/sprotty.git"
-  },
-  "dependencies": {
-    "@vscode/codicons": "^0.0.25",
-    "autocompleter": "5.1.0",
-    "file-saver": "2.0.2",
-    "inversify": "^5.0.1",
-    "snabbdom": "^3.0.3",
-    "tinyqueue": "^2.0.3"
+  "private": true,
+  "scripts": {
+    "prepare": "lerna run prepare",
+    "clean": "lerna run clean",
+    "build": "lerna run build",
+    "test": "lerna run test"
   },
   "devDependencies": {
-    "@types/chai": "^4.2.2",
-    "@types/file-saver": "^2.0.1",
-    "@types/jsdom": "^16.2.13",
-    "@types/mocha": "^5.2.7",
-    "@typescript-eslint/eslint-plugin": "^4.25.0",
-    "@typescript-eslint/parser": "^4.25.0",
-    "chai": "^4.2.0",
-    "circular-dependency-plugin": "^5.2.0",
-    "core-js": "^3.2.1",
-    "css-loader": "^3.2.0",
-    "eslint": "^7.27.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-no-null": "^1.0.2",
-    "file-loader": "^6.2.0",
-    "http-server": "^0.12.3",
-    "jenkins-mocha": "^8.0.0",
-    "jsdom": "^15.1.1",
-    "mocha": "^6.2.0",
-    "reflect-metadata": "^0.1.13",
-    "rimraf": "^3.0.0",
-    "semver": "^6.3.0",
-    "snabbdom-to-html": "^5.1.1",
-    "source-map-loader": "^0.2.4",
-    "style-loader": "^1.0.0",
-    "ts-loader": "^6.1.0",
-    "ts-mocha": "^8.0.0",
-    "ts-node": "^8.3.0",
-    "typescript": "3.8.3",
-    "webpack": "^4.39.3",
-    "webpack-cli": "^3.3.8"
-  },
-  "scripts": {
-    "clean": "rimraf lib artifacts",
-    "build": "tsc -p ./tsconfig.json && yarn run lint",
-    "lint": "eslint -c configs/.eslintrc.js \"src/**/!(*.spec.ts*|test-helper.ts)\"",
-    "test:cli": "ts-mocha \"./src/**/*.spec.?(ts|tsx)\"",
-    "watch": "tsc -w -p ./tsconfig.json",
-    "test": "jenkins-mocha --opts ./configs/mocha.opts \"./src/**/*.spec.?(ts|tsx)\"",
-    "prepare": "yarn run clean && yarn run build",
-    "examples:build": "webpack --config ./configs/webpack.config.js",
-    "examples:watch": "webpack --watch --config ./configs/webpack.config.js",
-    "examples:run": "http-server examples",
```

Comment:
It might be useful to still keep those scripts. It may help newcomers to identify more easily what to try.

Reply:
They are now in a separate package in the `examples` folder.

Reply:
We should document that in the README.